### PR TITLE
Upgrade to GitHub Provider 5.34.0

### PR DESCRIPTION
**Pull request by shivam-sharma7**

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
See letest provider: https://registry.terraform.io/providers/integrations/github/latest/docs/resources/actions_environment_secret

**Comment by shivam-sharma7** [comment]
@gastaldi I don't know why CI failing, any help would be apreciated

**Comment by shivam-sharma7** [comment]
okay

**Comment by shivam-sharma7** [comment]
@gastaldi see 

```
shivamsharma@shivam:/mnt/c/Users/shivam sharma/Desktop/Open source project/quarkiverse-devops/terraform-scripts$ terraform init

Initializing the backend...
╷
│ Error: Required token could not be found
│
│ Run the following command to generate a token for app.terraform.io:
│     terraform login

File: terraform-scripts/.terraform.lock.hcl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/integrations/github" {
-  version     = "5.33.0"
+  version     = "5.34.0"
```
Comment (gastaldi) [inline terraform-scripts/.terraform.lock.hcl:5]: You can't edit this manually, this is generated by the `terraform init --upgrade` command

**Review by gastaldi — CHANGES_REQUESTED** [review]
Unfortunately we have a limited number of allowed users and I'm afraid this must be done by someone with admin access to run `terraform init --upgrade`.

Is there any specific reason why you want the upgrade?

**Comment by gastaldi** [comment]
@shivam-sharma7 I've force-pushed what changed with `terraform init --upgrade`

**Review by gastaldi — APPROVED** [review]

**Comment by gastaldi** [comment]
Done, thanks for the update!

**Pull request merged**